### ISSUE TITLE
Test loss does not decrease in the training process

### ISSUE THREAD

**Issue by flak1990** [baidu/DuReader#34]
When i ran the BiDAF model (origin code without any modification) in the tensorflow version,  the training loss decreases to about 2. However, if we ran a dev mode after each epoch, the dev loss does not varies (always about 15). The bleu4 score is only about 20-25 in the dev set.  
i have also tried to add dropout in the training process, and to add l2 regularization in the training process, embedding matrix has also been replaced by a pretrained embedding model, but all of them have less effect on the result of the dev process. Dev loss still does not change in the dev process after each epoch.   
In addtion, the dataset is the training and dev set of baidu_search total dataset.   
Is there anything special we could do to train the tensorflow version and get a better result?

**Comment by lkliukai** [comment]
We believe that it is something wrong with your experiment, and did you follow our instruction in [https://github.com/baidu/DuReader/blob/master/README.md](url)?
Could you please provide your training steps in order to better analysis the situation?
